includes/admin/class-amp-post-meta-box.php

```diff
@@ -422,4 +423,22 @@ public function preview_post_link( $link ) {
 
 		return $link;
 	}
+
+	/**
+	 * Hide the `amp_status` custom field when edition a post.
+	 *
+	 * @param bool   $protected Whether the key is considered protected.
+	 * @param string $meta_key Metadata key.
+	 * @param string $meta_type Type of object metadata is for.
+	 * @return bool
+	 */
+	public function hide_amp_status_meta( $protected, $meta_key, $meta_type ) {
+		global $pagenow;
+
+		if ( is_admin() && 'post.php' === $pagenow && self::STATUS_POST_META_KEY === $meta_key && 'post' === $meta_type ) {
```

Comment:
I don't think we should filter this only on a single admin screen. For example, if someone uses  [`the_meta()`](https://developer.wordpress.org/reference/functions/the_meta/) in their theme to  output the movie attributes from their movie collection post type, we shouldn't show the AMP status either.
```suggestion
		if ( self::STATUS_POST_META_KEY === $meta_key && 'post' === $meta_type ) {
```

Reply:
@schlessera I've decided to go for a more robust way of solving this issue. I've added a summary of the changes above in the OP.

Reply:
Maybe instead of `is_admin()` then the check should be for `! REST_REQUEST`, since the point is to only expose it in the REST API.